packages/radix-vue/src/shared/component/Arrow.vue

```diff
@@ -39,6 +39,8 @@ const { primitiveElement } = usePrimitiveElement()
     :height="height"
     viewBox="0 0 30 10"
     preserveAspectRatio="none"
+    :as-child="props.asChild"
+    :as="as"
```

Comment:
let's remove this props assigment as we have `v-bind` to bind that props already 😁

Reply:
Yeah, I forgot to clean up properly